components/scream/src/physics/shoc/tests/shoc_brunt_length_tests.cpp

```diff
@@ -0,0 +1,121 @@
+#include "catch2/catch.hpp"
+
+//#include "share/scream_types.hpp"
+#include <algorithm>
+#include <array>
+#include <random>
+#include <thread>
+
+#include "ekat/scream_kokkos.hpp"
+#include "ekat/scream_pack.hpp"
+#include "ekat/scream_types.hpp"
+#include "ekat/util/scream_arch.hpp"
+#include "ekat/util/scream_kokkos_utils.hpp"
+#include "ekat/util/scream_utils.hpp"
+#include "physics/share/physics_constants.hpp"
+#include "physics/shoc/shoc_functions.hpp"
+#include "physics/shoc/shoc_functions_f90.hpp"
+#include "shoc_unit_tests_common.hpp"
+
+namespace scream {
+namespace shoc {
+namespace unit_test {
+
+TEST_CASE("shoc_brunt_length", "shoc") {
```

Comment:
In order to prepare these for BFB testing, please use the pattern that I did for calc-vertflux. Each unit test file should have a struct that has static methods for property and bfb tests.

Reply:
Thanks @jgfouca I completely missed that!  All *.cpp files in relation to this PR have been updated for your review.  I realize all other .cpp testing files (besides your calc-vertflux) are still stuck in the old pattern and need to be updated.  Figure maybe I should do those and update them in this PR, unless you prefer I do that in a separate PR?

Reply:
@bogensch , whatever is easiest for you.